app/serializers/node_info_serializer.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+class NodeInfoSerializer
```

Comment:
use ActiveModelSerializer

Reply:
Excuse my ignorance but how might I use one here?  Everywhere in mastodon there seems to be a pattern like this
```ruby
render json: @some_active_record_model, serializer: NodeInfoSerializer
```
But I don't have a particular active record model I am rendering.